File: Beecrowd/Python/ex1015.py
def distancia_pontos(ab1, od1, ab2, od2):
    """
    :param ab1: abcissa1 (x1)
    :param od1: ordenada1 (y2)
    :param ab2: abcissa2 (x2)
    :param od2: ordenada2 (y2)
    :return: Distãncia entre dois pontos no plano.
    """
    import math
    distancia = math.sqrt((ab2-ab1)**2+(od2-od1)**2)
    return distancia


entrada_1 = input()
x1, y1 = [float(z) for z in entrada_1.split()]
entrada_2 = input()
x2, y2 = [float(z) for z in entrada_2.split()]

print(f'{distancia_pontos(x1, y1, x2, y2):.4f}')

File: Beecrowd/C/ex1015.c
#include <stdio.h>
#include <math.h>

float distanciaPontos(float P1[], float P2[]);

int main() {
    float P1[2], P2[2];
    scanf("%f", &P1[0]);
    scanf("%f", &P1[1]);
    scanf("%f", &P2[0]);
    scanf("%f", &P2[1]);

    printf("%.4f", distanciaPontos(P1, P2));

    return 0;
}

float distanciaPontos(float P1[], float P2[]) {
    float x1 = P1[0];
    float y1 = P1[1];
    float x2 = P2[0];
    float y2 = P2[1];
    float distancia = sqrtf(powf(x2-x1, 2)+powf(y2-y1, 2));

    return distancia;
}
